mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule FileStore.MixProject do
     [
       app: :file_store,
       version: @version,
-      elixir: "~> 1.14",
+      elixir: "~> 1.15",
```

Comment:
I'm not opposed to this, but what's the benefit of dropping 1.14 support?

Reply:
It does not get active support anymore. Its end of life is coming soon. https://endoflife.date/elixir

Typically the support range should be the last 3 version of elixir and 3 versions of OTP. You also won't find people using Elixir 1.14 and OTP27. So it cuts down on the possible permutations of support vs OTP support.